python/cugraph/tests/dask/opg_pagerank_testing.py

```diff
@@ -38,10 +34,33 @@ def test_dask_pagerank():
                              names=['src', 'dst', 'value'],
                              dtype=['int32', 'int32', 'float32'])
 
+    df = cudf.read_csv(input_data_path,
+                       delimiter=' ',
+                       names=['src', 'dst', 'value'],
+                       dtype=['int32', 'int32', 'float32'])
+
     g = cugraph.DiGraph()
-    g.from_dask_cudf_edgelist(ddf)
+    g.from_cudf_edgelist(df, 'src', 'dst')
+
+    dg = cugraph.DiGraph()
+    dg.from_dask_cudf_edgelist(ddf)
+
+    # Pre compute local data
+    # dg.compute_local_data(by='dst')
```

Comment:
Is this comment meant to be removed?

Reply:
This is only to be used for benchmarking so we pre-compute local data. Otherwise it is not required to be run prior to calling the algorithm